### ISSUE TITLE
Missing page

### ISSUE THREAD

**Issue by markobr** [UB-Mannheim/Reichsanzeiger#4]
Nr. 35/1875: First page of Central-Handelsregister is missing.

**Comment by stweil** [comment]
Yes, and a lot of other pages are missing or misplaced, too. If you are lucky, you might find the missing page somewhere else because it was only misplaced.

The digital edition is based on a microfilm edition which has several quality issues. UB Mannheim tries to fix those by making new scans to replace bad microfilm images or add missing pages. This process takes some time and is currently finished for the years 1920 to 1945 (see https://digi.bib.uni-mannheim.de/periodika/zeitungen/zeitungenlistenansicht/).
